src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public Rectangle(int firstLeg, int secondLeg) {
```

Comment:
Consider adding a constructor with a 'color' parameter for more flexibility when creating a Rectangle instance 🌈